Whac-A-Mole.md

```diff
@@ -0,0 +1,60 @@
+# Summary
+> A task to show that imitation exists in competitive settings.
```

Comment:
Players were required to hit targets on a large visual touchscreen before their competitors.